projects/packages/jetpack-mu-wpcom/webpack.config.js

```diff
@@ -49,6 +48,10 @@ module.exports = [
 		},
 		resolve: {
 			...jetpackWebpackConfig.resolve,
+			alias: {
+				...jetpackWebpackConfig.resolve.alias,
+				'@automattic/calypso-config': '@automattic/calypso-config/src/client.js',
```

Comment:
Just curious, why do we need that?

Reply:
The [default entry](https://www.npmjs.com/package/@automattic/calypso-config?activeTab=code) has server-side code (e.g.: path) and it causes the error during the build. As a result, we have to specify the client.js manually